scripts/docker-eth

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+mkdir -p ~/.ethereum ~/.web3
+
+# check if sudo is necessary
+if ! id -nG $(whoami)|grep -qw "docker"; then SUDO='sudo'; fi
```

Comment:
If you assume that the variable was not previously set, that is, but they are inherited from the environment.